spec/system/admin/settings_spec.rb

```diff
@@ -65,33 +59,14 @@
                                        'on "Features" tab.'
     end
 
-    scenario "Should show successful notice" do
+    scenario "Should update marker" do
```

Comment:
Since we have a commit to group tests "Group similar map tests together", do you think it would make sense to also group the tests "Should be able when map feature activated" (lines 50-60) and "Should update marker" (lines 62-82)?